shared/reducers/fs.js

```diff
@@ -155,6 +167,44 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
         ]
       }, [])
       return state.mergeIn(['pathItems'], resetsToMerge)
+    case FsGen.newFolderRow:
+      const {parentPath} = action.payload
+      const parentPathItem = state.pathItems.get(parentPath, Constants.makeUnknownPathItem())
+      if (parentPathItem.type !== 'folder') {
+        console.warn(`bad parentPath: ${Types.pathToString(parentPath)}`)
```

Comment:
we might not want to log people's paths

Reply:
We already log paths in KBFS logs though. But I guess I can remove the path here.